T3/App/ImGuiDx11RenderForm.cs

```diff
@@ -17,7 +17,7 @@ public class ImGuiDx11RenderForm : RenderForm
         public ImGuiDx11RenderForm(string title)
             : base(title)
         {
-            MouseMove += (o, e) => ImGui.GetIO().MousePos = new System.Numerics.Vector2(e.X, e.Y);
+            //MouseMove += (o, e) => ImGui.GetIO().MousePos = new System.Numerics.Vector2(e.X, e.Y);

```

Comment:
Please avoid commented code without explaining comment while it's still relevant.
In this case removing it would have been just fine.